app/services/bike_finder.rb

```diff
@@ -25,7 +25,7 @@ def self.find_matching(serial:, owner_email:)
       .where(serial_normalized: SerialNormalizer.new(serial: serial).normalized)
       .where(
         "bikes.owner_email = ? OR ownerships.user_id IN (?) OR ownerships.creator_id IN (?)",
-        owner_email,
+        email,
```

Comment:
Good catch! Why don't any of the `BikeFinder` specs catch this?

Reply:
Updated 👍  b56df9b8